src/components/board/components/styles.module.scss

```diff
@@ -62,3 +62,18 @@
   flex-direction: column;
   gap: 5px;
 }
+
+.owner-select {
+  height: 24px;
+  appearance: none;
+  border: 0;
+  outline: none;
+  font-size: var(--font-size-200);
+  padding: 3px 0;
+  cursor: pointer;
+}
+
+.owner-select:hover,
+.owner-select option:hover .owner-select {
```

Comment:
What should this selector select?

Reply:
Owner?🤔

Reply:
You cannot select an item higher than what you have already selected. If your selector looks like '.owner-select option:hover .owner-select' it will select .owner-select inside option if you hover it, not a select element itself